server/src/test/java/io/crate/metadata/upgrade/MetadataIndexUpgraderTest.java

```diff
@@ -58,7 +59,7 @@ public void testDynamicStringTemplateIsPurged() throws IOException {
     @Test
     public void test__all_is_removed_from_mapping() throws Throwable {
```

Comment:
Do we still need this? At which version did we introduce the `_all` migration?

Reply:
I think you are referring to this? 9680445ca3385515f364546090f416427c74c8ef
This has been in effect since 4.0.7. Since the `minimumIndexCompatibility` is now 4.0.0, looks like we should keep it